alert-common/src/main/java/com/synopsys/integration/alert/common/persistence/model/ConfigurationFieldModel.java

```diff
@@ -89,12 +85,8 @@ public void setFieldValues(final Collection<String> values) {
         return Collections.emptySet();
     }
 
-    public void setSet(final boolean set) {
-        isSet = set;
-    }
-
     public boolean isSet() {
-        return isSet;
+        return fieldValues != null;
```

Comment:
Is checking if the collection is empty an appropriate check that should be included?